src/main/java/net/minecraftforge/common/ForgeConfig.java

```diff
@@ -82,29 +82,34 @@ public static class Server {
     /**
      * General configuration that doesn't need to be synchronized but needs to be available before server startup
      */
+    @Deprecated(since = "1.19", forRemoval = true)
```

Comment:
This is being deprecated in 1.19.2 (and same comment for the rest of the deprecations in here).
```suggestion
    @Deprecated(since = "1.19.2", forRemoval = true)
```